### ISSUE TITLE
"Run All" command in REPL

### ISSUE THREAD

**Issue by srush** [zed-industries/zed#16961]
### Check for existing issues

- [X] Completed

### Describe the feature

It will be very useful to have a "Run All" feature in the REPL that runs all the cell blocks individually. You can currently highlight everything but that only runs as if it was a big cell. In my use case I would like to see all the results aligned with the cells of the code.

**Comment by haznai** [comment]
I'd also love a restart + run all option. It's easy to forget in which state your application/cell is, so just cleaning the state and re-running everything is killer ux.

i'd argue that things like [marimo](https://github.com/marimo-team/marimo) exist just because jupyter kernels have this ambigous state problem.

awesome job so far on the repl though, i really like my new workflow :)

**Comment by ermaolaoye** [comment]
Agreed, very indeed wanting a Run All feature for the REPL environment.

**Comment by gepcel** [comment]
Seems that not many people use zed with python. Are you still intrested in this feature? Would you mind leaving some comments on [Proposal 46594](https://github.com/zed-industries/zed/discussions/46594),thank you. @srush @zihang-huang @haznai
